spec/lib/msf/ui/console/command_dispatcher/creds_spec.rb

```diff
@@ -5,6 +5,11 @@
 require 'msf/ui/console/command_dispatcher/creds'
 
 RSpec.describe Msf::Ui::Console::CommandDispatcher::Creds do
+
+  if ENV['REMOTE_DB']
+    before {skip("Awaiting credentials port")}
+  end
```

Comment:
@mkienow-r7 This might be a bit of a grave dig; But I don't suppose you know the latest information on this `skip` message? 😄 

Were the creds API ever ported to the REMOTE_DB?